src/PublishCodeCovCoverageTask/__tests__/webUtils.test.ts

```
@@ -0,0 +1,879 @@
+        // Restore original implementation
+        jest.spyOn(require('../utils/webUtils'), 'downloadFile').mockImplementation(originalDownloadFile);
+    }, 10000);
+
+    test('should report download progress', async () => {
+        // Setup mocks
+        const writeStream = {
+            on: jest.fn().mockImplementation(function (this: any, event, handler) {
+                if (event === 'finish') setTimeout(handler, 0);
+                return this;
+            }),
+            close: jest.fn((cb) => cb && cb())
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)